packages/opencode/src/session/processor.ts

```
@@ -146,20 +246,24 @@ export namespace SessionProcessor {
                   break
 
                 case "tool-input-start":
+                  // altimate_change start — sanitize the incoming id before it
+                  // becomes the persisted callID and the pairing key.
+                  const inputStartCallID = coerceToolCallID(value.id)
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)